### ISSUE TITLE
How to freeze parameters when training?

### ISSUE THREAD

**Issue by speshowBUAA** [open-mmlab/mmdetection#5614]
I want to add a new branch to a well-pretrained model and preserve the parameters of the original model. How to set "requires_grad  = False" in an elegant way?

**Comment by RangiLyu** [comment]
Currently, only backbones support freeze parameters. If you want to stop gradients in heads and necks, you can implement the `_freeze_stages` in those parts. May refer to this https://github.com/open-mmlab/mmdetection/blob/5af50631d92bcf2a7841e997c48bd515d2980b5b/mmdet/models/backbones/resnet.py#L572
